Mqtt-Plugin

grosj opened the pull request.

grosj committed:
Configuration for mqtt

grosj committed:
Plugin for mqtt

grosj committed:
Changelog für MQTT

grosj committed:
Update CHANGELOG.md

grosj commented:
Läuft bereits seit über einer Woche erfolgreich bei mir.

Schrolli91 commented on plugins/mqtt/mqtt.py:
Wieso return an dieser Stelle?

grosj replied:
In der Zeile davor stand vorher ein Logging, dass dieser Typ nicht supported wird. Beim Einfügen der Config-Abfrage habe ich dann vergessen das return zu entfernen. Ist nachgebessert.

Schrolli91 commented on plugins/mqtt/mqtt.py:
Wieso return an dieser Stelle?

grosj replied:
In der Zeile davor stand vorher ein Logging, dass dieser Typ nicht supported wird. Beim Einfügen der Config-Abfrage habe ich dann vergessen das return zu entfernen. Ist nachgebessert.

Schrolli91 commented on plugins/mqtt/mqtt.py:
Vielleicht wäre es sinnvoll, den Port auch noch konfigurierbar zu machen? Was meinst du?

grosj replied:
Mhh... könnte man machen... Werd ich mal einbauen. Dann muss es allerdings bei mir nochmal probelaufen.

grosj committed:
Returns entfernt

flothi commented:
Ich fände es super, wenn wir direkt die verschlüsselte Verbindung zum Broker anbieten würden. Sprich Zertifikat hinterlegen.

grosj committed:
Update mqtt.py

- ClientName über Config konfigurierbar
- Port über Config konfigurierbar
- Zertifikate über Config konfigurierbar

grosj committed:
Update config.template.ini

- ClientName über Config konfigurierbar
- Port über Config konfigurierbar
- Zertifikate über Config konfigurierbar

grosj commented:
Hab mal ein paar Konfigurationsmöglichkeiten hinzugefügt.
Läuft bei mir bisher einwandfrei... Allerdings habe ich verschlüsselte Verbindung inkl. der Zertifikate nicht getestet.

Schrolli91 commented:
Ich versuch die Woche mal genauer drüber zu schauen - bin aktuell etwas eingespannt.
Wenn du magst, kannst du ja mal versuchen über das BWCC den ein oder anderen zu einem Test zu motivieren :-D

grosj commented:
> Ich versuch die Woche mal genauer drüber zu schauen - bin aktuell etwas eingespannt.
> Wenn du magst, kannst du ja mal versuchen über das BWCC den ein oder anderen zu einem Test zu motivieren :-D

Jepp, alles gut...

Schrolli91 commented:
@grosj gabs nen Grund für die Löschung?

grosj commented:
Ich hab einfach mal aufgeräumt. Hatte gedacht, dass es bereits im development-Branch drin ist. Da war ich wohl zu voreilig.
